opencog/persist/zmq/atomspace/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ INSTALL (TARGETS zmqatoms
 )
 
 INSTALL (FILES
-	ZMQMessages.pb.h
+	${CMAKE_CURRENT_BINARY_DIR}/ZMQMessages.pb.h
```

Comment:
thanks.

I sometuimes think we are installing too many header files.  Many headers are really "internal use only"...